server/AccManagementTest.java

```diff
@@ -0,0 +1,82 @@
+package newbank.server;
+
+import static org.junit.Assert.*;
+
+import newbank.client.TestClient;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+
+public class AccManagementTest {
+    TestClient client = new TestClient();
+
+    @Before
+    //starts by logging in
+    public void setUp() throws IOException {
+        client.startConnection("localhost", 14002);
+        client.loginMessage();
+    }
+
+    @After
+    //stops connection after each test to reset back to main menu
+    public void complete() throws IOException {
+        client.stopConnection();
+    }
+
+    @Test
+    //Test1 - test showAccounts`
+    public void test1() throws IOException {
```

Comment:
As we talked on the meeting, names of the tests are important, and should follow some structure to make them more readable.

I'd suggest something like this:
showAccounts_returns_validResponse
showAccounts_returnsResponse

This goes on for all the tests in the file